sherpa/astro/ui/tests/test_astro_ui_unit.py

```diff
@@ -640,18 +640,18 @@ def test_save_data_data2d(tmp_path):
 
     # the output depends on the backend, and neither seems ideal
     #
-    if backend.__name__ == 'sherpa.astro.io.crates_backend':
+    if io.backend.__name__ == 'sherpa.astro.io.crates_backend':
         expected = ["#TEXT/SIMPLE", "# X0 X1 Y SHAPE"]
         s = [2, 3, 0, 0, 0, 0]
         for xi, yi, zi, si in zip(x, y, z, s):
             expected.append(f"{xi} {yi} {zi} {si}")
 
         expected = "\n".join(expected) + "\n"
 
-    elif backend.__name__ == 'sherpa.astro.io.pyfits_backend':
+    elif io.backend.__name__ == 'sherpa.astro.io.pyfits_backend':
         expected = "\n".join([str(zz) for zz in z]) + "\n"
     else:
-        raise RuntimeError(f"UNKNOWN I/O BACKEND: {backend.__name__}")
+        raise RuntimeError(f"UNKNOWN I/O BACKEND: {io.backend.__name__}")
```

Comment:
I guess this could just be

```
assert False, io.backend.__name__
```

but it's not really worth changing.

Reply:
I'm just following the previous convention here, wither is fine.